references/obj_detection/train_pytorch.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+import os
+
+os.environ['USE_TORCH'] = '1'
+
+import time
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+
+from doctr.datasets import DocArtefacts
+from model import faster_model
+from utilities import *
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+
+def fit_one_epoch(model_, images, targets):
```

Comment:
one epoch means all batches from the dataset, but this function only take one batch :sweat_smile:

Reply:
Thanks, changes has been made :) Yeah earlier, I executed it inside the dataloader. It is removed now:)